app/javascript/components/base/dots-dropdown.vue

```diff
@@ -43,7 +43,17 @@ export default {
       active: false,
     };
   },
+  mounted() {
+    document.addEventListener('click', this.closeDropdown);
+  },
+  unmounted() {
+    document.removeEventListener('click');
```

Comment:
Acá tienes que pasarle la función tbn
```suggestion
    document.removeEventListener('click', this.closeDropdown);
```

Reply:
oops